docs/ci/jenkins.md

```diff
@@ -202,8 +202,8 @@ When options can be set at both levels, project settings override global setting
         <td>Include Sauce Connect output in the Jenkins console output for each job.</td>
       </tr>
       <tr>
-        <td><b>Launch Sauce Connect On Slave</b></td>
-        <td>Launch Sauce Connect on the Jenkins slave node instead of the master node.</td>
+        <td><b>Launch Sauce Connect On Node</b></td>
+        <td>Launch Sauce Connect on the a Node instead of the Master Server.</td>
```

Comment:
```suggestion
        <td>Launch Sauce Connect on a Node instead of the Master Server.</td>
```

Reply:
Does Jenkins also have an alternative term to replace Master Server?

Reply:
I believe they just call it the Jenkins Server